docs/cloudinit.adoc

```diff
@@ -6,33 +6,21 @@
 :toc: auto
 
 :uri-cloudinit: https://docs.oracle.com/en-us/iaas/Content/ContEng/Tasks/contengusingcustomcloudinitscripts.htm
+:uri-source-cloudinit-doc: https://github.com/oracle-terraform-modules/terraform-oci-oke/blob/main/docs/instructions.adoc#configuring-cloud-init-for-the-nodepools
 
-== Instructions:
+== Configuring Cloud init scripts for Nodepools:
+Cloud init scripts passed to the node pools can be passed in 2 different ways.
 
-* {uri-cloudinit}[Using cloud-init in OKE] 
-* The default cloud-init script used by this module is as below:
-----
-#!/bin/bash
-
-# DO NOT MODIFY
-curl --fail -H "Authorization: Bearer Oracle" -L0 http://169.254.169.254/opc/v2/instance/metadata/oke_init_script | base64 --decode >/var/run/oke-init.sh
-
-## run oke provisioning script
-bash -x /var/run/oke-init.sh
+* *Using `cloudinit_nodepool_common`*: When a script is passed to the cloudinit_nodepool_common, this init doc is executed in all the node pools.
```

Comment:
Elaborate on each under their own sections e.g. how to customize the common script

Reply:
Added examples for both types.